src/main/java/de/digitalcollections/streaming/euphoria/config/SpringConfigWeb.java

```diff
@@ -50,4 +54,25 @@ public LocaleResolver localeResolver() {
     localeResolver.setDefaultLocale(Locale.ENGLISH);
     return localeResolver;
   }
+
+  @Bean
+  public Map<String, String> webjarVersions() {
+    Map<String, String> versions;
+    Map<String, String> customVersions;
```

Comment:
customVersions is only used inside the second try-with-resources block and could be defined there.